Assets/Scripts/Game/TalkManager.cs

```diff
@@ -533,24 +543,20 @@ int GetReactionToPlayer_0_1_2(QuestionType qt)
             if (toneIndex == 0)
             {
                 skillValue = player.Skills.GetLiveSkillValue(DFCareer.Skills.Etiquette);
+                toneModifier += etiquetteReactionMods[socialGroup];
                 if (toneReactionForTalkSession[0] == 0)
                     player.TallySkill(DFCareer.Skills.Etiquette, 1);
             }
             else if (toneIndex == 2)
             {
                 skillValue = player.Skills.GetLiveSkillValue(DFCareer.Skills.Streetwise);
+                toneModifier += streetwiseReactionMods[socialGroup];
                 if (toneReactionForTalkSession[2] == 0)
                     player.TallySkill(DFCareer.Skills.Streetwise, 1);
             }
 
-            // Make roll result be the same every time for a given NPC.
-            if (currentNPCType == NPCType.Mobile)
-                DFRandom.Seed = (uint)lastTargetMobileNPC.GetHashCode();
-            else if (currentNPCType == NPCType.Static)
-                DFRandom.Seed = (uint)lastTargetStaticNPC.GetHashCode();
-
             if (toneIndex != 1)
-                toneModifier = DFRandom.random_range_inclusive(1, 100) >= skillValue ? -5 : 10;
+                toneModifier += DFRandom.random_range_inclusive(1, 100) >= skillValue ? -10 : 5;
```

Comment:
You can use Dice100.FailedRoll(skillValue) ? -10 : 5 here

Reply:
Good catch 👍 Done.